src/vignette/util/services.clj

```diff
@@ -1,10 +1,12 @@
 (ns vignette.util.services
-  (:require [vignette.util.consul :as consul]
-            [clojure.string :refer [join]]
-            [environ.core :refer [env]]))
+  (:require [clojure.string :refer [join]]
+    [environ.core :refer [env]]))
+
 
 (defn materialize-static-asset-url [oid]
-  (str
-    (consul/->uri
-      (consul/find-service
-        consul/create-consul "static-assets" consul/service-query-tag)) "/image/" oid))
\ No newline at end of file
+  (let [environment (env :wikia-environment)
+        physical-dc (env :wikia-datacenter)
+        logical-dc (if (= environment "dev") (clojure.string/join "-" [physical-dc environment]) physical-dc)]
+    (if (empty? environment) (throw (Exception. "WIKIA_ENVIRONMENT not set"))
```

Comment:
I think it'd be better to move this check to a function and then create constants for `environment` and `logical-dc`, so that the exception is thrown earlier.